lib/reporters/index.js

```diff
@@ -18,10 +18,17 @@ internals.protos = {
     tap: require('./tap')
 };
 
+internals.requireReporter = function (reporter) {
+    if (reporter.indexOf('.') === 0) {
```

Comment:
our style is to add a newline when we create new scopes: http://hapijs.com/styleguide